_posts/2018-04-01-from-chemical-engineering-to-software-development.md

```diff
@@ -0,0 +1,117 @@
+# From chemical engineering to software development
+
+1st of April, 2016, Paris, _Gare du Nord_. With my backpack and my monitor in a bag, I onboard a Eurostar train to London with my cousin Tom and his wife Emma. I left a chemical engineering position behind me, for a software developer one.
+
+The year is now 2018, I work for a company called [Driftrock](https://www.driftrock.com/): we boost the marketing power of our clients through social media. I'm a software developer there, taking care of an awesome microservices fleet with a heavily-skilled team and juggling between back-end, front-end and DevOps, using various programming languages to achieve our goals. Before that, I had spent more than a year at [The App Business](http://www.theappbusiness.com/), a tech company with a specialisation in mobile technologies. There I was a junior software engineer, learning key concepts like Agile, testing and clean code.
+
+Time for a check-in after 2 years, isn't it?
+
+I want to share here some insights on my career switch. Personal opinions and experiences are exposed, in the most honest way possible, far be it from me to bash the chemical engineering industry, nor force any chemical engineers to shift away from their jobs! I am still keen to take any feedback and comments if you disagree.
+
+Unfortunately, I give here only the differences between a non-tech industry and the software industry, therefore, the purpose of this article is not about how to start your software engineer career (however, you can still drop me a message if you are looking for any pieces of advice).
+
+The differences between the British and the French way of working are not tackled here, it is a wide subject deserving a full and proper article.
+
+## Where it all began...
+
+With few words, the `Once upon a time` is: I grew up on a French island ([Reunion Island](https://www.google.co.uk/maps/place/Reunion/@-21.1351121,55.2471124,10z/data=!3m1!4b1!4m5!3m4!1s0x2178778110b8e43b:0x4a7f8e89ecdbeaf9!8m2!3d-21.115141!4d55.536384) - oversea territory), did a normal scholarship until my _baccalauréat_ (UK GCSE equivalent), attended to intensive courses in mathematics, physics and chemistry for 2 years (_Classe Preparatoire aux Grandes Ecoles_), was admitted into a national chemical engineering school ([ENSIC](https://ensic.univ-lorraine.fr/)) in the lovely town of Nancy (the Lorraine region, in the mainland), graduated after 3 years and found my first job as a chemical engineer in Paris.
+
+From that point, the career path was almost sealed: I would have spent the next decades in the same company, painfully risen through the ranks, fought against this silly bureaucracy where seniority often results in adopting a more managerial role and becoming removed from technical processes - hence become disconnected from technological advances, and then gone in retirement at 70 years old with nothing (stay lucid, this is going to happen).
```

Comment:
Just one more small correction!
- forgot 'ing' at the end of 'become' i.e `becoming more disconnected from technological advances` (or tbh you could remove this last part entirely)
- and possibly you could also remove 'with nothing' :)

Reply:
Ah thanks @emilywoods I've lightened a bit 🦅 : 44e0279

Just to let you know, a friend of mine was reading, and asked this very interesting question:
> I am curious, did you find it hard learning to write code and understand all the tech jargon or was it already an interest therefore quite easy to pick up?

That's a good damn question for both us, but I'm afraid it's off-topic here 🤔